tests/ui_tests/test_cluster_list_page.py

```diff
@@ -638,3 +638,66 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterConfigPage:
+    @pytest.mark.usefixtures("create_community_cluster")
```

Comment:
As I see, all tests inside the class use it, so it'll be better to move it to class level

Reply:
i add one without this fixture

Reply:
Got it